developer-workflow/psrt.rst

```diff
@@ -0,0 +1,148 @@
+Python Security Response Team (PSRT)
+====================================
+
+The Python Security Response Team (PSRT) is responsible for handling
+vulnerability reports for CPython and pip.
+
+Vulnerability report triage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Vulnerability reports are sent to one of two locations,
+the long-standing ``security@python.org`` mailing list
+or using the private vulnerability reporting feature
+of GitHub Security Advisories.
+
+For reports sent to ``security@python.org``, a PSRT admin
+will triage the report and if the report seems plausible
+(ie, not spam and for the correct project) will reply with
+instructions on how to report the vulnerability on GitHub.
+
+If the reporter doesn't want to use GitHub's Security Advisories feature
+then the PSRT admins can create a draft report on behalf of the reporter.
+
+Coordinating a vulnerability report
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each report will have a member of the PSRT assigned as the "coordinator".
+The coordinator will be responsible for following the below process and
+will be publicly credited on vulnerability records post-publication.
+
+If a coordinator can't complete the process for any reason (time obligation,
+vacation, etc) they must find a replacement coordinator in the PSRT
+and reassign the vulnerability report appropriately.
+
+Coordinators are expected to collaborate with other PSRT members and core developers
+when needed for guidance on affect-ness, severity, advisory text, and fixes.
+
+**The vulnerability coordination process is:**
+
+* Determine whether the report constitutes a vulnerability. If the report isn't a vulnerability,
+  the reporter should be notified appropriately. Close the GHSA report, the report can be reopened if
+  sufficient evidence that the report is a vulnerability is later obtained.
+
+* After a vulnerability report is accepted, a CVE ID must be assigned. If this is not done
+  automatically, then a CVE ID can be obtained by sending an email to ``cna@python.org``.
+  No details about the vulnerability report need to be shared with the PSF CNA for a CVE ID to be reserved.
```

Comment:
I feel like we should emphasise here that this is the coordinator's job, not a researcher's job (e.g. "then the coordinator should send an email to ..." in the middle bit). The last sentence in particular makes it sound like something a researcher might do before contacting the PSRT.

Reply:
Resolved in https://github.com/python/devguide/pull/1348/commits/0344b7ce0b59c9ecebb41e28f353ed67ab704b30